app/public/static/js/gallery_carousel.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
```

Comment:
needs licence header

```suggestion
// @author recluse4615
// @license magnet:?xt=urn:btih:1f739d935676111cfff4b4693e3816e664797050&dn=gpl-3.0.txt GPL-v3-or-Later

"use strict";
```